src/Commands/pm/SecurityUpdateCommands.php

```diff
@@ -154,14 +154,12 @@ protected function calculateSecurityUpdates($composer_lock_data, $security_advis
      * Packages are discovered via composer.lock file. An exit code of 3
      * indicates that the check completed, and insecure packages were found.
      *
-     * Thanks to https://github.com/FriendsOfPHP/security-advisories
-     * and Symfony for providing this service.
-     *
      * @param array $options
      *
      * @return UnstructuredData
      * @throws \Exception
      * @command pm:security-php
+     * @validate-php-extension ext-zip
```

Comment:
Composer won't allow installation at all if the zip extension isn't installed will it?

https://github.com/drush-ops/drush/blob/10.x/composer.lock#L1077

Reply:
True, but one can use --ignore-platform-reqs to bypass.

Reply:
Its too bad Enlightn bakes in this Zipfile cache. Would be better to factor that out as an optional dependency.